### PR TITLE
codeintel: run tests in ci

### PR DESCRIPTION
**Pull request by davejrt**

### CODE REVIEW

**Review by efritz — APPROVED** [review]

File: internal/cmd/precise-code-intel-tester/main.go [hunk 1]
```diff
@@ -7,11 +7,18 @@ import (
...
 
 	"github.com/sourcegraph/sourcegraph/internal/env"
+	"github.com/sourcegraph/sourcegraph/internal/gqltestutil"
 )
 
+var client *gqltestutil.Client
+
 var (
-	endpoint = env.Get("ENDPOINT", "http://127.0.0.1:3080", "Sourcegraph frontend endpoint")
-	token    = env.Get("ACCESS_TOKEN", "", "Access token")
```
Comment (davejrt) [inline internal/cmd/precise-code-intel-tester/main.go:14]: @efritz these were changed largely so they are more inline with what we currently use in other tests. It makes it a little easier to integrate with the existing test pipelines we have.

Comment (efritz) [reply internal/cmd/precise-code-intel-tester/main.go:14]: Yeah no problem, standardization will (usually) always get a 👍 from me.

Comment (efritz) [reply internal/cmd/precise-code-intel-tester/main.go:14]: Let me know if this utility should be moved somewhere else (I see a `/test` directory I don't remember being familiar with).

**Comment by davejrt** [comment]
closed #14665

**Pull request merged**